.editorconfig

```diff
@@ -3,8 +3,15 @@ root = true
 [*]
 charset = utf-8
 end_of_line = lf
-indent_size = 4
+indent_size = 2
 indent_style = space
-insert_final_newline = false
+insert_final_newline = true
 max_line_length = 120
+tab_width = 2
+
+[*.{kt,kts}]
+indent_size = 4
 tab_width = 4
```

Comment:
👍  We may also want to add .java files here as well as there is likely to be some Java files in this repo :)

Reply:
added here. https://github.com/inngest/inngest-kt/pull/13